pages/api/Schema/reports/fourteenMonth/fourteenMonth.graphql

```diff
@@ -35,6 +35,23 @@ type FourteenMonthReportCurrencyGroupTotalsMonth {
   total: Float!
 }
 
+enum StatusEnum {
```

Comment:
Is there a way we could import this instead of manually redefining it? This would be nice if the statues ever changed again.

Reply:
Unfortunately, since the proxy GraphQL server and the Rails GraphQL are completely separate servers there's not a way that I know of to share enums or types between them.

Reply:
Ok gotcha